src/librustc/Cargo.toml

```diff
@@ -13,12 +13,12 @@ arena = { path = "../libarena" }
 fmt_macros = { path = "../libfmt_macros" }
 graphviz = { path = "../libgraphviz" }
 log = "0.3"
+owning_ref = "0.3.3"
```

Comment:
@rkruppe your argument re: LLVM etc makes sense to me. I'm curious to hear what @brson has to say since I consider him more the expert on this stuff than me. =)

(In general, though, I'm excited about gaining the ability to use external crates in the compiler.)